source/index.md

```diff
@@ -492,22 +522,80 @@ POST https://gateway.clearhaus.com/authorizations
     <i>Optional</i> <br />
     A reference to an external object, such as an order number.
   </dd>
+
+  <!-- deprecated -->
   <dt>threed_secure[pares]</dt>
-  <dd>[:base64:] <br /> <i>Optional</i> <br /> See more information on <a target="_blank" href="http://docs.3dsecure.io">3Dsecure.io</a></dd>
+  <dd>
+    Deprecated! Please use <code>card[pares]</code>. <br />
+    [:base64:] <br />
+    See more information at <a target="_blank" href="http://docs.3dsecure.io">3Dsecure.io</a>.
+  </dd>
   <dt>card[number]</dt>
+  <dd>
+    Deprecated! Please use <code>card[pan]</code>. <br />
+    [0-9]{12,19} <br />
+    Primary account number of card to charge.
+  </dd>
+</dl>
+
+##### Method: `card`
+
+<dl class="dl-horizontal">
+  <dt>card[pan]</dt>
   <dd>[0-9]{12,19} <br /> Primary account number of card to charge.</dd>
   <dt>card[expire_month]</dt>
   <dd>[0-9]{2} <br /> Expiry month of card to charge.</dd>
   <dt>card[expire_year]</dt>
   <dd>20[0-9]{2} <br /> Expiry year of card to charge.</dd>
   <dt>card[csc]</dt>
   <dd>[0-9]{3} <br /> Card Security Code.</dd>
+  <dt>card[pares]</dt>
+  <dd>[:base64:] <br /> <i>Optional</i> <br /> See more information at <a target="_blank" href="http://docs.3dsecure.io">3Dsecure.io</a></dd>
 </dl>
 
 <p class="alert alert-info">
-<b>Notice:</b> The card dictionary is not required when making an authorization
-based on a card resource. However, <code>card[csc]</code> should be populated
-when CSC is available.
+  <b>Notice:</b> An authorization that includes <code>card[pares]</code> is 3-D
+  Secured and is considered multi-factor authenticated.
```

Comment:
Maybe phrase this a bit differently in order to account for unauthenticated, attempt and error PARes'?

Reply:
"An *approved* authorization that ..."?

Reply:
:+1: